processor/removeemptyvaluesprocessor/config.go

```diff
@@ -0,0 +1,75 @@
+// Copyright  observIQ, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package removeemptyvaluesprocessor provides a processor that removes empty values from telemetry data
+package removeemptyvaluesprocessor
+
+import (
+	"bytes"
+	"fmt"
+)
+
+// valid fields that can be referenced in the MapKey's field
+const (
+	attributesField = "attributes"
+	resourceField   = "resource"
+	bodyField       = "body"
+)
+
+// MapKey represents a key into a particular map (denoted by field)
+type MapKey struct {
+	field string
+	key   string
+}
+
+// UnmarshalText unmarshals the given []byte into a MapKey.
+// The format of the key is "<field>.<path-to-key>"
+func (m *MapKey) UnmarshalText(text []byte) error {
+	field, key, found := bytes.Cut(text, []byte("."))
```

Comment:
Can I specify `body`, `resource`, or `attributes` to ignore all of that.  This seems to indicate no but I can also see specifying `body` in the exclude is the same as `enable_log_body = false`.

Reply:
This is more just putting the thought out there I realize it doesn't work this way.

Reply:
Yeah, this might be a more elegant way to handle it than these boolean values. I'll rework it to work like that, and remove the booleans.